Core/GDCore/Extensions/Builtin/KeyboardExtension.cpp

```diff
@@ -82,6 +82,16 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsKeyboardExtension(
                     "res/conditions/keyboard.png")
       .AddCodeOnlyParameter("currentScene", "");
 
+ extension
+      .AddCondition("AnyKeyReleased",
+                    _("Any key Released"),
```

Comment:
`Any key released`

(careful about capitalization)

Reply:
Oh yes excuse me. I am french (like you) so I put capital at begining of a sentence.

Reply:
I am correcting it !

Reply:
Done

Reply:
Oh crap I erased the head by accident :/ I think it's not a big deal

Reply:
It's not a problem. Prefer batching your answer into a single one because otherwise I'm spammed with notifications. Thanks :)